src/shared/components/VideoCategoryCard/VideoCategoryCard.style.ts

```diff
@@ -0,0 +1,180 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+import { transparentize } from 'polished'
+
+import { colors, sizes, transitions } from '@/shared/theme'
+
+import { VideoCategoryCardProps } from '.'
+import { Text } from '../Text'
+
+type ColorProps = { color: string }
+type LoadingProps = { loading?: VideoCategoryCardProps['loading'] }
+type VariantProps = { variantCategory?: VideoCategoryCardProps['variant'] }
+export const CoverImg = styled.div<{ bgImgUrl: string }>`
+  background: url(${({ bgImgUrl }) => bgImgUrl});
+  background-position: center;
+  background-size: cover;
+  filter: grayscale(100%);
```

Comment:
Why are we using a filter instead of a color overlay like the design does?

Reply:
not needed anymore the overlay is there